src/OpenTK/Platform/Windows/WinRawKeyboard.cs

```diff
@@ -107,7 +107,7 @@ public void RefreshDevices()
                         string deviceClassGUID = (string)regkey.GetValue("ClassGUID"); // for windows 8 support via OpenTK issue 3198
 
                         // making a guess at backwards compatability. Not sure what older windows returns in these cases...
-                        if(deviceClass == null || deviceClass.Equals(string.Empty)){
+                        if (deviceClass == null || deviceClass.Equals(string.Empty)){
```

Comment:
Probably a separate PR but can we do something about braces at end of lines like this one?